native/src/main/java/bblfsh/bash/Main.java

```diff
@@ -0,0 +1,18 @@
+package bblfsh.bash;
+
+public class Main {
+
+    public static void main(String args[]) {
+        final RequestReader reader = new RequestReader(System.in);
```

Comment:
Maybe is better to instance a unique ObjectMapper object and send it to the RequestReader and ReponseWriter over the constructor.